src/caselawclient/models/utilities/aws.py

```diff
@@ -146,3 +151,40 @@ def notify_changed(uri: str, status: str, enrich: bool = False) -> None:
         Subject=f"Updated: {uri} {status}",
         MessageAttributes=message_attributes,
     )
+
+
+def copy_assets(old_uri: str, new_uri: str) -> None:
+    """
+    Copy *unpublished* assets from one path to another,
+    renaming DOCX and PDF files as appropriate.
+    """
+    client = create_s3_client()
+    bucket = env("PRIVATE_ASSET_BUCKET")
+    old_uri = uri_for_s3(old_uri)
+    new_uri = uri_for_s3(new_uri)
+
+    response = client.list_objects(Bucket=bucket, Prefix=old_uri)
+
+    for result in response.get("Contents", []):
+        old_key = str(result["Key"])
+        new_key = build_new_key(old_key, new_uri)
+        if new_key is not None:
```

Comment:
we can reduce nesting here:

```suggestion
        if new_key is None:
            continue
```

and then unident the code block that is currently in the if statement